addons/skin.confluence/720p/DialogPVRGuideOSD.xml

```diff
@@ -203,7 +219,15 @@
 						<width>20</width>
 						<height>20</height>
 						<texture>PVR-HasTimer.png</texture>
-						<visible>ListItem.HasTimer + !ListItem.HasTimerSchedule + !ListItem.IsRecording</visible>
+						<visible>ListItem.HasTimer + !ListItem.HasTimerSchedule + !ListItem.IsRecording + ListItem.IsEnable</visible>
+					</control>
+					<control type="image">
+						<left>120</left>
+						<top>5</top>
+						<width>20</width>
+						<height>20</height>
+						<texture>PVR-HasTimer-Disabled.png</texture>
+						<visible>ListItem.HasTimer + !ListItem.HasTimerSchedule + !ListItem.IsRecording + !ListItem.IsEnabled</visible>
 					</control>
```

Comment:
Maybe a SetProperty(Timerstate) or sush? 
Then we could name them:
"" -> no timer
TimerEnabled
TimerDisabled
TimerError
TimerRecording
, ....
And just add .png after that string to get our icon, then a whole bunch of duplicate code can be dropped.

What do you think? Or are properties not meant for that?